src/WebJobs.Script.Grpc/Channel/GrpcWorkerChannel.cs

```diff
@@ -528,9 +528,10 @@ internal void SendFunctionLoadRequest(FunctionMetadata metadata, ManagedDependen
 
         internal FunctionLoadRequest GetFunctionLoadRequest(FunctionMetadata metadata, ManagedDependencyOptions managedDependencyOptions)
         {
+            var functionId = metadata.GetFunctionId();
```

Comment:
nit: if this is `string`, please us language types over `var`
```suggestion
            string functionId = metadata.GetFunctionId();
```

Reply:
have reverted this change